src/widgets/search-box/search-box.js

```diff
@@ -186,18 +185,12 @@ function searchBox({
         // handle IE8 weirdness where BACKSPACE key will not trigger an input change..
         // can be removed as soon as we remove support for it
         if (INPUT_EVENT === 'propertychange' || window.attachEvent) {
-          addListener(input, 'keyup', ifKey(KEY_SUPPRESS, getInputValueAndCall(setQuery)));
           addListener(input, 'keyup', ifKey(KEY_SUPPRESS, getInputValueAndCall(maybeSearch)));
```

Comment:
Reading the comment saying that we will remove this when we drop IE8 support, isn't this already the case?